discussions/tasks.py

```diff
@@ -67,4 +97,21 @@ def sync_channel_memberships():
         log.debug('OPEN_DISCUSSIONS_USER_SYNC is set to False (so disabled) in the settings')
         return
 
-    api.sync_channel_memberships()
+    # establish when we'll stop syncing this "batch"
+    end_time = time.time() + SYNC_MEMBERSHIPS_LOCK_TTL_SECONDS
```

Comment:
Can you change this to `now_in_utc() + timedelta(seconds=SYNC_MEMBERSHIPS_LOCK_TTL_SECONDS)`? And similarly below in `should_sync`?